server/controllers/chartsController/chartsController.test.js

```diff
@@ -47,59 +51,64 @@ describe('chartsController', () => {
   })
   describe('index', () => {
     describe('When successful', () => {
-      it('returns a list of charts and a status of 200', async () => {
-        const request = createRequestWithUser()
-        const response = createResponse()
-        const user = createUser()
-        const chart = createChart({
-          assessment: 'EEG Quick',
-          _id: 'chart_eeg_id',
-          variable: 'Rating',
-          owner: 'user',
+      beforeAll(async () => {
+        dataDb = await global.MONGO_INSTANCE.db('dpdata')
+        appDb = await global.MONGO_INSTANCE.db('appDb')
+      })
+
+      beforeEach(async () => {
+        await dataDb.createCollection('charts')
+      })
+      afterEach(async () => {
+        await dataDb.collection('charts').drop()
+      })
+
+      it('returns a list of charts in order of user favorite, title, alongside a status of 200', async () => {
+        const [chart1, chart2, chart3] = [
+          createChart({
+            assessment: 'EEG Quick',
+            _id: ObjectId(),
```

Comment:
Do we need the ID or will it be set automatically?

Reply:
If we need it maybe build it into the fixture

Reply:
[e39a67e](https://github.com/AMP-SCZ/dpdash/pull/557/commits/e39a67eb3ef48faac109401996dd3d299b3fdc6b)